src/js/utils/SchemaFormUtil.js

```diff
@@ -117,7 +117,7 @@ let SchemaFormUtil = {
         return;
       }
 
-      let nextDefinition = definition.definition.find(
+      let nextDefinition = [].concat.apply([], definition.definition).find(
```

Comment:
Let's be a little more verbose here and write `Array.prototype.concat.apply` this is not only easier to read but also don't create an unneeded instance of array. 

BTW:  Be aware that `concat.apply([],[[[1]]])` will return `[[1]]` not  `[1]`. So please make sure we only have on level of nested arrays?